go/ias/init.go

```diff
@@ -0,0 +1,59 @@
+// Package ias implements the IAS proxy client.
+package ias
+
+import (
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/oasislabs/oasis-core/go/common/identity"
+	"github.com/oasislabs/oasis-core/go/common/logging"
+	"github.com/oasislabs/oasis-core/go/common/sgx/ias"
+	"github.com/oasislabs/oasis-core/go/ias/api"
+	"github.com/oasislabs/oasis-core/go/ias/proxyclient"
+	cmdFlags "github.com/oasislabs/oasis-core/go/oasis-node/cmd/common/flags"
+)
+
+const (
+	CfgProxyAddress       = "ias.proxy_addr"
+	CfgTLSCertFile        = "ias.tls"
+	CfgDebugSkipVerify    = "ias.debug.skip_verify"
+	CfgAllowDebugEnclaves = "ias.debug.allow_debug_enclaves"
```

Comment:
Should those be shared with oasis-node? Maybe also put it behind `dont_blame_oasis`?

Reply:
Yeah, these should be `dont_blame_oasis` options.